src/components/FloatingFlow.vue

```diff
@@ -168,16 +168,34 @@ export default {
       mainTabName: "Flatmap",
       zIndex: 1,
       showDialogIcons: false,
-      dockedArray: [{title: "Flatmap", id:1}],
+      dockedArray: [{title: "Flatmap", id:1}, {title: "Heatmap", id:2},  {title: "Plot", id:3}],
```

Comment:
I do not think we should have all three dialog docked right at the beginning. It will destroy the flow of the current use case. This can be added in the fetchContent method in the MapPopover module instead.